packages/typir/src/kinds/class-kind.ts

```diff
@@ -108,7 +264,7 @@ export class ClassKind implements Kind {
 
     isSubType(superType: Type, subType: Type): TypirProblem[] {
         if (isClassKind(superType.kind) && isClassKind(subType.kind)) {
-            if (this.options.structuralTyping) {
+            if (this.options.typing === 'Structural') {
```

Comment:
Structural/Nominal can be two default implementations off class-kind, I think.
The `isSubType` decides if it is nominal or structural, and this could differ a bit from DSL to DSL.

Reply:
Yes, that would be the alternative.